msteams-platform/concepts/build-and-test/apps-package.md

```diff
@@ -16,7 +16,7 @@ You need an app package however you plan to distribute your Microsoft Teams app.
 
 When a user installs your app in Teams, they install an app package that contains only a configuration file (also known as an app manifest) and your app's icons. The app's logic and data storage are hosted elsewhere, such on localhost during development and Azure Web Services. Teams accesses these resources via HTTPS.
 
-:::image type="content" source="../../assets/images/teams-app-host.png" alt-text="Illustration showing app hosting for Teams app":::
+:::image type="content" source="../../assets/images/teams-app-host.png" alt-text="The screenshot is an example for the illustration that shows app hosting for Teams app.":::
```

Comment:
Just read this again `The screenshot is an example for the illustration`.

Reply:
resolved